### PR TITLE
Docs: git-repo structure update to add valuesFiles and label reference

### PR DESCRIPTION
**Pull request by ibrokethecloud**
Update for git repo structure to show how to use 

* valuesFiles
* cluster labels as variable values

### CODE REVIEW

File: docs/gitrepo-structure.md [hunk 1]
```diff
@@ -69,6 +69,13 @@ helm:
   # install.
   values:
     any-custom: value
+  # All labels on Rancher clusters are available using global.fleet.clusterLabels.LABELNAME
+  # These can now be access directly as variables
```
Comment (nickgerace) [inline docs/gitrepo-structure.md:73]: nit `accessed`

**Review by nickgerace — APPROVED** [review]

**Review by StrongMonkey — APPROVED** [review]

**Comment by StrongMonkey** [comment]
We should hold on getting docs PR merged until we push the full release that has these features

File: docs/gitrepo-structure.md (hunk 1 above)
Comment (ibrokethecloud) [reply docs/gitrepo-structure.md]: thanks for the picking this up..

**Review by nickgerace — APPROVED** [review]

**Comment by nickgerace** [comment]
Reminder to all (because I almost merged it): https://github.com/rancher/fleet/pull/330#issuecomment-816792795

**Review by catherineluse — APPROVED** [review]

**Pull request merged**